Formula/kvazaar.rb

```diff
@@ -1,25 +1,31 @@
 class Kvazaar < Formula
-  desc "HEVC encoder"
+  desc "Ultravideo HEVC encoder"
   homepage "https://github.com/ultravideo/kvazaar"
-  url "https://github.com/ultravideo/kvazaar/archive/v0.5.0.tar.gz"
-  sha256 "2facdbffcf739171127487cd7d1e48c925560f39755a16542c4a40e65e293070"
-
-  bottle do
-    cellar :any_skip_relocation
-    sha256 "a50a695243ea971c8ab2cb85358f3095128e52be588df7b609cbf22048fb58ef" => :el_capitan
-    sha256 "1cf96786d4613d94b93a8091027ea9d14ae6c801d11996def7c4b408316032e6" => :yosemite
-    sha256 "aa78c8fa2657ccac25a0c3ac30d7840f281ab8af8a59aafb8058a122090e1b97" => :mavericks
-    sha256 "5fc3c49c10479539474539e1bee928f89a3586f1f44e2f346c3b6484ff6395d2" => :mountain_lion
-  end
```

Comment:
oh, ok though I feel I should still drop ` -    cellar :any_skip_relocation` since the build process has improved a lot lately